web/static/js/components/voting_lower_third_content.jsx

```diff
@@ -5,14 +5,28 @@ import StageProgressionButton from "./stage_progression_button"
 
 import * as AppPropTypes from "../prop_types"
 
-const VotingLowerThirdContent = props => (
-  <div className="ui stackable grid basic attached secondary center aligned segment">
-    <VotesLeft currentUser={props.currentUser} />
-    <div className="five wide computer eight wide tablet column">
-      <StageProgressionButton {...props} />
+const VotingLowerThirdContent = props => {
+  const facilitorComponent = (
+    <div className="ui stackable grid basic attached secondary center aligned segment">
+      <div className="two wide column">
+        <VotesLeft currentUser={props.currentUser} />
+      </div>
+      <div className="four wide column">
+        <StageProgressionButton {...props} />
+      </div>
     </div>
-  </div>
-)
+  )
+
+  const nonFacilitorComponent = (
```

Comment:
hey @tnewell5, here we're repeating almost all of the markup for the component, where the only difference between facilitator and non-facilitator interfaces is the column containing the stage progression button. to keep this DRY, you could use the short-circuit operator, something like so:

```es6
<div className="ui stackable grid basic attached secondary center aligned segment">
  <div className="two wide column">
    <VotesLeft currentUser={props.currentUser} />
  </div>
  { props.currentUser.is_facilitator && 
    <div className="four wide column">
      <StageProgressionButton {...props} />
    </div>
   }
</div>
```

Reply:
done!